tests/test_kernels/triton/test_token_attn_1.py

```diff
@@ -0,0 +1,104 @@
+import math
+
+import torch
+
+from colossalai.kernel.triton.token_attention_kernel import token_attn_fwd_1
+
+
+def torch_attn(xq, xk, bs, seqlen, num_head, head_dim):
```

Comment:
tests follows this format? https://github.com/hpcaitech/ColossalAI/pull/4485/files#diff-232c19b679c915358eb4e7a5066a4110c9b2c7a7aca467597966e4d6fe8e9818R23

Reply:
done.